crates/bitwarden/src/error.rs

```diff
@@ -94,6 +94,16 @@ pub enum EncStringParseError {
     InvalidLength { expected: usize, got: usize },
 }
 
+impl From<chrono::ParseError> for Error {
+    fn from(e: chrono::ParseError) -> Self {
+        Self::Internal(string_to_static_str(e.to_string()))
+    }
+}
+
+fn string_to_static_str(s: String) -> &'static str {
+    Box::leak(s.into_boxed_str())
+}
```

Comment:
Not perfect. Maybe we should just drag in the chrono parse errors as a valid subclass of errors.

Reply:
We've had a couple of instances where it was annoying to have a `&'static str` as the internal error. We could just change it to `String`, the tiny overhead of the allocation in the error path is not a big deal.

We already have serde and base64 errors, I think it would make sense to add chrono there as a separate error as well. We can consider moving them all into a separate ParsingError to avoid cluttering the main Error enum if it's a big issue.